### PR TITLE
Add error details on UNHANDLED_ERROR_THROWN events

### PR DESCRIPTION
**Pull request by forabi**

### CODE REVIEW

File: src/app/index.tsx [hunk 1]
```diff
@@ -89,10 +95,9 @@ window.addEventListener(
     store.dispatch(
       unhandledErrorThrown({
         location: pick(location, 'pathname', 'search', 'hash'),
-        message:
-          typeof reason === 'object' && 'message' in reason
-            ? reason.message
-            : String(reason),
+        name: 'Unhandled Rejection',
+        message: String(reason),
+        ...(isError(reason) ? reason : undefined),
```
Comment (wholesomedev) [inline src/app/index.tsx:100]: Wouldn't this try to spread `undefined`?

**Review by wholesomedev — COMMENTED** [review]
LGTM

File: src/app/index.tsx (hunk 1 above)
Comment (forabi) [reply src/app/index.tsx:100]: Yes, and it's valid :smile: 

One weirdness in JavaScript:

```
{ ...undefined } // OK
[...undefined] // TypeError
```

Comment (wholesomedev) [reply src/app/index.tsx:100]: TIL

**Pull request merged**